fuzzing_tc/common/pool.py

```diff
@@ -376,6 +273,8 @@ def cycle_crons(self):
         """
         if self.schedule_start is not None:
             now = self.schedule_start
+            if now.utcoffset() is not None:
```

Comment:
I guess we could just unconditionally convert to UTC here.

Reply:
This just checks if any timezone was given in the date string. If none was given, `utcoffset()` will return `None` and we'll treat it as UTC already.

Reply:
OK, I just tried it.

If `utcoffset()` is called on a naïve `datetime` (with no timezone), it does return `None`, but if `astimezone(UTC)` is called on that `datetime` object, it is assumed to be in the *local* timezone. That's not what we want.

What we want is to treat the `datetime` as already being UTC, which means calling `replace(tzinfo=UTC)`. I'll change that so it's explicit that we're always working in UTC.